esyi/bin/esyi.re

```diff
@@ -1,4 +1,4 @@
-open EsyInstaller;
+open EsyInstall;
```

Comment:
The `EsyInstall` and `esyi` names are a bit confusing (as `esyi` is short for `esy installer`) - but I don't have a better suggestion.

Reply:
Yes, the overall situation with “install” used both for solving/fetching dependencies and for installation of built artifacts is confusing...